charters/DocumentationCharter.md

```diff
@@ -0,0 +1,54 @@
+# Docs Working Group Charter (Proposed)
+
+## Mission and Goals
+
+The Docs Working Group serves as a support, helper, and advisory body to Jupyter subprojects on all aspects of documentation. The core pillars of our mission:
+
+- Improve all aspects of documentation across the Jupyter ecosystem
+- Make high quality documentation that is clear, comprehensive, inclusive, and serves the varying needs of Jupyter's diverse community
+- Engage with the community to help users with Jupyter products, discover their needs, and connect them with relevant information, expertise and resources
+
+The Docs Working Group will provide a place for consistent, focused, holistic efforts to be spent on docs across the whole ecosystem. This group exists in part to provide capacity and resources to the subprojects (some of which are already suffering from a lack of resources/capacity, and more specifically to work on docs in particular).
+
+We want all users to have positive experiences inside the Jupyter ecosystem, especially users who are learning and coming in for the first time, and users with disabilities.
+
+## The Docs Working Group's Activities
+
+The Docs Working Group will focus on the efforts described below, in service of its mission:
+
+- Help write docs (inside the bounds of each subproject's governance)
```

Comment:
One question that I have is that Jupyter has some documentation that isn't (currently) part of subprojects. Will this WG own that documentation? Right now this type of documentation is in the grey area of the project where it isn't clear who can even review/merged PRs. I think it would be helpful if the charter gave the WG ownership of those docs.